### PR TITLE
Add 2015-07-14-securestring

### PR DESCRIPTION
**Pull request by terrajobst**

### CODE REVIEW

File: 2015-07-14-securestring/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Secure String
+
+## Problems with `SecureString`
+
+* The purpose of `SecureString` is to avoid having secrets store in the process
```
Comment (akoeplinger) [inline 2015-07-14-securestring/README.md]: ~~store~~ stored

File: 2015-07-14-securestring/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Secure String
```
Comment (akoeplinger) [inline 2015-07-14-securestring/README.md]: No link to the YouTube recording?

**Comment by akoeplinger** [comment]
Left some comments, LGTM otherwise.

File: 2015-07-14-securestring/README.md (hunk 2 above)
Comment (terrajobst) [inline 2015-07-14-securestring/README.md]: Ah, good one. Let me fix that.

File: 2015-07-14-securestring/README.md (hunk 1 above)
Comment (terrajobst) [inline 2015-07-14-securestring/README.md]: Goo catch! :eyeglasses:

File: 2015-07-14-securestring/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# API Review 2015-07-14
+
+This API review was also recorded and is available on [Google+](https://plus.google.com/events/cms40br51f6vt7eg73grju99690).
+
+## Overview
+
+In this API review we reviewed we discussed what we do with `SecureString`
```
Comment (akoeplinger) [inline 2015-07-14-securestring/README.md]: ~~we reviewed~~ we discussed

File: 2015-07-14-securestring/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+* In general, there is a move away from using passwords
+    - Ideally, we'd have an abstraction to store a reference to a means for an
+      authenticated user
+    - APIs would then take the abstraction as opposed to individual pieces of
+      of information, such as user name / password.
+
+### Conclusion
+
+* We probably shouldn't expose `SecureString` in .NET Core
+* The only incoming API dependency is `Process`. The primary consumer of the the
```
Comment (akoeplinger) [inline 2015-07-14-securestring/README.md]: ~~the~~ the

**Comment by akoeplinger** [comment]
I'll refrain from proofreading now, I promise :sunglasses:

File: 2015-07-14-securestring/README.md (hunk 3 above)
Comment (terrajobst) [inline 2015-07-14-securestring/README.md]: Fixed

File: 2015-07-14-securestring/README.md (hunk 4 above)
Comment (terrajobst) [inline 2015-07-14-securestring/README.md]: Fixed

**Comment by terrajobst** [comment]
No worries. Feedback is feedback! :-)

**Pull request merged**